src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'categories/CHECK_STATUS';
+
+const INITIAL_STATE = 0;
```

Comment:
- [ ] Kindly set the `INITIAL_STATE` with an empty state as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Set%20the%20initial%20state%20to%20be%20an%20empty%20array%20of%20categories.).

Reply:
Thank you for your feedback. I have implemented the requested change and set the INITIAL_STATE with an empty state as required. 😄